lib/client/commands/login.go

```diff
@@ -116,5 +140,30 @@ func (l *Login) Run(cmd *cobra.Command, args []string) error {
 		}
 	}
 
+        // TODO(isaac): wrap below in a function and call via defer
+        // will that still execute if the user ctrl-c's ?
+        grpclog.SetLoggerV2(grpclog.NewLoggerV2(os.Stdout, os.Stderr, os.Stderr))
+        grpc.EnableTracing = true
+        conn, err = grpc.Dial("localhost:8981", grpc.WithInsecure(), grpc.WithUnaryInterceptor(tokenAuthInterceptor(enCreds.Token)))
```

Comment:
You probably want to add grpc.WithTimeout(2 * time.Second).

Reply:
done